src/engine/devdraw.c

```diff
@@ -1236,7 +1237,9 @@ GdDrawImageByPoint(PSD psd, MWCOORD x, MWCOORD y, PMWIMAGEHDR pimage)
 
 	GdFixCursor(psd);
 }
+#endif /* MW_FEATURE_IMAGES*/
```

Comment:
the corresponding `#if` was commented out, so either this endif is completely dangling or at least the comment no longer accurate.

Reply:
Thanks for looking at this. I think the diff combined with my change makes this looks odd - as there used to be an embedded \#if MW_FEATURE_IMAGES on line 851 which I changed to \#if MW_FEATURE_ALPHA and commented out.

So, this \#endif and comment are actually correct and match the corresponding \#if MW_FEATURE_IMAGES on line 533.